cdap-docs/application-templates/source/etl/templates/sources/real-time/index.rst

```diff
@@ -11,6 +11,5 @@ Sources: Real-time
 
     JMS <jms>
     Kafka <kafka>
-    SQS <sqs>
```

Comment:
We'll need to talk to Albert. He removed it in this PR (https://github.com/caskdata/cdap/pull/3365), though I see that the java file (...p-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/SqsSource.java) was still in the PR.

https://github.com/caskdata/cdap/pull/3365/files#diff-a584fe18788dda704f586682d79f70fd

Probably needs to be restored and re-vised, unless there is some issue with exposing it.